### PR TITLE
2 lab rpi

### PR DESCRIPTION
**Pull request by OmazSleer**

### CODE REVIEW

File: first.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width-device-width, initial-scale-1.0"/>
```
Comment (OmazSleer) [inline first.html]: Chrome dev tools : device preview isn't working

File: first.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+</head>
+<body>
+	<header>
+		<div id="small-menu">
+			<p>Navigation</p>
+			<a href="#"><i class="material-icons">dehaze</i></a>
+		</div>
+		<nav id="head-menu">
+			<div>
```
Comment (OmazSleer) [inline first.html]: ul > li > a

File: first.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+				<a href="#"> PSD WEBSITES </a>
+				<a href="#"> UI KITS</a>
+				<a href="#"> PREMIUM PSD FILES</a>
+				<a href="#"> ABOUT</a>
+				<a href="#"> BLOG</a>
+		    </div>
+	    </nav>
+	</header>
+	<div class="banner">
+		<p id="name-ban">Awesome Free PSD </br>templates & Resources</p>
```
Comment (OmazSleer) [inline first.html]: h1 instead of p

File: first.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+		<div id="www">www.BlazRobar.com</div>
```
Comment (OmazSleer) [inline first.html]: h2/h3/span/p

**Comment by aliaksandrkrasnou** [comment]
Approved, good 👍

**Pull request closed without merging**